src/libs/ajax.js

```diff
@@ -170,6 +171,31 @@ const User = signal => ({
       `${await Config.getTosUrlRoot()}/user/response`,
       _.mergeAll([authOpts(), { signal, method: 'POST' }, jsonBody({ ...tosData, accepted: true })])
     )
+  },
+
+  createSupportRequest: async ({ name, email, currUrl, subject, type, description }) => {
+    const options = {
+      method: 'POST',
+      body: JSON.stringify({
+        request: {
+          requester: { name, email },
+          subject,
+          'custom_fields': [
+            { id: 360012744452, value: type },
```

Comment:
Are we going to have different id's on dev and prod?

Reply:
Yes, these are IDs are from Zendesk. I don't think that they will change, they are specific to our organization's Zendesk but I will look into it.

Reply:
From what I can tell, the IDs won't change as long as we don't change the custom fields (add or subtract). If so, then we'll need to add them here.

Reply:
Sorry I missed your question @zarsky-broad, no we won't have different ID's on dev and prod. Right now all tickets sent from either dev or prod go to the same Zendesk portal. We do have a sandbox environment we could use but it would require replicating the full setup of what Comms has done. I can do this but it would take at least a day. 

@panentheos do you think it is worth the time to set up the Zendesk sandbox for our dev testing?

Reply:
(Wasn't saying we needed to, just recalled something to that effect and thought I'd ask.)

Reply:
As far as I can tell from the [Zendesk developer documentation about custom fields](https://developer.zendesk.com/rest_api/docs/core/tickets#setting-custom-field-values), the only way to populate the custom fields is by ID. 
Only Admin-level users within Zendesk can edit the custom fields, which is limited to Tiffany and Kate Noblett and myself, though my permissions will be downgraded soon. I let both of them know that if they make changes to the form on Zendesk that it will break our implementation.